src/backend/src/prisma-query-args/projects.query-args.ts

```diff
@@ -35,7 +35,8 @@ const projectQueryArgs = Prisma.validator<Prisma.ProjectArgs>()({
         expectedActivities: { where: { dateDeleted: null } },
         deliverables: { where: { dateDeleted: null } }
       }
-    }
+    },
+    favoritedBy: true
```

Comment:
not sure if were ever gonna need to know who has favorited it from a project perspective